### ISSUE TITLE
Error: print() has moved. Use require('sys') to bring it back.

### ISSUE THREAD

**Issue by jfhbrook** [gf3/Jerk#3]
with

```
[josh@eevee lib]$ node --version
v0.1.100-9-gb1901cd
```

got this:

```
node> require('./jerk')
Error: print() has moved. Use require('sys') to bring it back.
    at node.js:11:11
    at Function.bind (native)
    at new Jerk (/home/josh/lulzbot/lib/Jerk/lib/jerk.js:11:26)
    at Object.<anonymous> (/home/josh/lulzbot/lib/Jerk/lib/jerk.js:8:12)
    at Module._compile (module:424:23)
    at Module._loadScriptSync (module:434:8)
    at Module.loadSync (module:306:10)
    at loadModule (module:251:12)
    at require (module:374:12)
    at cwdRequire (repl:26:10)
```

Honestly, I have _no clue_ where to start with this one.

**Comment by jfhbrook** [comment]
I replaced the sys.print in js-IRC with console.log(sys.inspect()) but the same error is still there

MADDENING

**Comment by gf3** [comment]
This is actually a V8 bug in the native `Function#bind` implementation, here's the issues here: http://github.com/v8/v8/commit/fcfec20146c12d454347a6e0d50727a0477d57f2#commitcomment-117306

So we're just waiting for node/v8 to clean that up.

**Comment by jfhbrook** [comment]
Awesome! Thanks.

**Comment by gf3** [comment]
Fixed in ry/node@b2a2bb736b01dfcd909d2c9087135bc6cb3e2e15
